config/initializers/devise.rb

```diff
@@ -7,6 +7,7 @@
   # Devise will use the `secret_key_base` as its `secret_key`
   # by default. You can change it below and use your own secret key.
   # config.secret_key = '....'
+  config.secret_key = "ENV[DEVISE_SECRET_KEY]"
```

Comment:
Try setting in your secrets.yml - or setting in your environment.

We should probably set up figaro gem for this in the future.